tasks/main.yml

```diff
@@ -292,7 +278,101 @@
       debug:
         var: __mssql_sqlcmd_input_file.stdout_lines
 
-    - name: Remove the {{ __mssql_sql_tempfile.path }} file
+    - name: Remove the tempfile
       file:
         path: "{{ __mssql_sql_tempfile.path }}"
         state: absent
+
+- name: Configure the sqlagent setting
+  include_tasks: mssql_conf_setting.yml
+  vars:
+    __mssql_conf_setting: "sqlagent enabled"
+    __mssql_conf_setting_value: "{{ mssql_enable_sql_agent }}"
+  when: mssql_enable_sql_agent is not none
+
+- name: Install the {{ __mssql_server_fts_packages }} package
+  package:
+    name: "{{ __mssql_server_fts_packages }}"
+    state: "{{ 'present' if mssql_install_fts else 'absent' }}"
```

Comment:
If `mssql_install_fts` is other than `true`, `package` is not executed? I.e., there is no chance to have `state: absent`?

Reply:
Well, I say that `mssql_install_fts` must be of type bool. Hence this login. true = install, false = remove

Reply:
But, `when: false` (mssql_install_fts is set to false), this task won't be executed, will it? 
E.g., if you run this playbook,
```
  vars:
    mssql_install_fts: false

  tasks:
    - name: print
      debug:
        msg: "PRINTED - {{ mssql_install_fts }}"
      when: mssql_install_fts
```
this print task is skipped:
```
TASK [print] ***********************************************************************************************
skipping: [localhost]
```
So, there is no chance for the package to be removed. I'd think the condition has to be: `mssql_install_fts` is set (not none) and the boolean value is true or false?

Reply:
Yes I see what is the issue now, I should change `when: mssql_install_fts` to `when: mssql_install_fts is not none`.

Reply:
Done